examples/core/gated_gcn/train.py

```diff
@@ -1,6 +1,9 @@
+import time
```

Comment:
move it down, after the file description.

Reply:
OK, done.